tempesta_fw/t/unit/sched_helper.c

```diff
@@ -73,6 +73,12 @@ test_start_sg(TfwSrvGroup *sg, const char *sched_name, unsigned int flags)
 	kernel_fpu_end();
 
 	sg->flags = flags;
+	/* Adjust servers weights for static ratio scheduler. */
+	if (!strcmp(sched_name, "ratio")
+	    && (sg->flags & TFW_SG_F_SCHED_RATIO_STATIC))
```

Comment:
At configuration time, server weights are set/adjusted for any variation of Ratio scheduler, not just static. For Dynamic and Predictive variations these server weights are initial weights before any dynamic data is collected.